smokey/src/main/java/frc/robot/commands/DriveRampUpSpeedCommand.java

```diff
@@ -0,0 +1,64 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: DriveRampUpSpeedCommand.java
+// Intent: Forms a command to ramp up the drivetrain subsystem speed until it reaches maximum.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+
+public class DriveRampUpSpeedCommand extends CommandBase {
+
+    private DrivetrainSubsystem drivetrainSub;
+    private Timer timer = new Timer();
+    private final double powerIncrementDelaySeconds = 0.5;
+
+
+    public DriveRampUpSpeedCommand(DrivetrainSubsystem drivetrainSub) {
+        this.drivetrainSub = drivetrainSub;
+        // DO NOT declare drivetrainSub as requirement.  
+        // We need default drive commands to keep working while this command runs
+    }
+
+    /**
+     * Override of the initialize method on CommandBase - called on start of command
+     */
+    @Override
+    public void initialize() {
+        drivetrainSub.incrementPowerReductionFactor();
```

Comment:
This method will only increment it one time.  It won't return the robot to its original power (after say a RampDownSpeed), no?  Just seems like the intent of RampUp is to counteract/revert what RampDown did ... which I don't think is what is happening.  Maybe I'm missing something.

Reply:
I tested in simulation, and it does increment multiple times until the robot has reached max power.  
When you press the trigger, it ramps the power down until your each the minimum, holds at minimum, until you release the trigger, then it ramps back up to max.